docs/topics/aad.md

```diff
@@ -6,19 +6,77 @@ This walkthrough is to help you get start with Azure Active Directory(AAD) integ
 
 ## Prerequisites
 1. An Azure Active Directory tenant, referred to as `AAD Tenant`. You can use the tenant for your Azure subscription;
-2. A `Web app / API` type AAD application, referred to as `Server Application`. This application represents the `apiserver`. For groups to work properly, you'll need to edit the `Server Application` Manifest and set `groupMembershipClaims` to either `All` or `SecurityGroup`.
-3. A `Native` type AAD application, referred to as `Client Application`. This application is for user login via `kubectl`. You'll need to add delegated permission to `Server Application`, please see [troubleshooting](#loginpageerror) section for detail.
+2. Admin access to the Azure Active Directory Tenant
 
-You also need to delegate permission to the application as follows:
+> [!NOTE]
+> This feature is not supported on ADFS
 
-1. Go to Azure Portal, navigate to `Azure Active Directory` -> `App registrations`.
-2. Select the `Client Application`, Navigate to `Settings` -> `Required permissions`
-3. Choose `Add`, select the `Server Application`. You may need to enter the Server Application's name into the search field and search for it.
-   In permissions tab, select `Delegated permissions` -> `Access {Server Application}`
+## Create the Server Application on AAD
+
+An App Registration which serves as a resource identifier for the Kubernetes cluster. 
+
+1. On the Azure portal, select **Azure Active Directory** > **App registrations** > **New registration**.
+
+    a. Give the application a name, such as *KubernetesApiserver*.
+
+    b. For **Supported account types**, select **Accounts in this organizational directory only**.
+
+    c. Select **Register** when you're finished.
+
+
+2. Select **Manifest**, and then edit the **groupMembershipClaims:** value as **"All"**. When you're finished with the updates, select **Save**.
+
+
+3. In the left pane of the Azure AD application, select **Expose an API**, and then select **+ Add a scope**.
+
+    a. Enter a **Scope name**, an **Admin consent display name**, **Admin consent description**, **User consent display name** and **User consent description**.
+
+    b. Select **Who can consent** as **Admins and Users**.
+    > [!NOTE]
+    > **Admins and Users** setting will enable every user to provide consent on their behalf. If you want to restrict that you can choose **Admins only**. However it will require one time admin consent on app.
+
+    c. Make sure **State** is set to **Enabled**.
+
+    d. Select **Add scope**.    
+
+4. Return to the application **Overview** page and note the **Application (client) ID**. When you deploy an OpenID enabled Kubernetes cluster with AAD integration, this value is called the server application ID (serverAppID).
+
+## Create the Client Application on AAD
+
+The second App Registration is used when you sign in with the Kubernetes CLI (kubectl).
```

Comment:
Should we clarify that the purpose for this credential is to trigger the AAD device auth process? Probably also add a link to what the AAD device sign in is? https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-device-code